src/utils.js

```diff
@@ -79,16 +79,19 @@ module.exports = {
 		return new Promise((resolve, reject) => {
 			function getResponse() {
 				// Present the user with a prompt for input.
-				res.reply(prompt);
+				robot.emit('ibmcloud.formatter', { response: res, message: prompt});
+
 
 				// Control the response when the timeout expires.
 				dialog.dialogTimeout = function(msg) {
 					/* istanbul ignore next */
-					res.reply(i18n.__('conversation.timed.out'));
+					robot.emit('ibmcloud.formatter', { response: res, message: i18n.__('conversation.timed.out')});
 				};
 
 				// Handle the expected response.
 				dialog.addChoice(regex, (msg) => {
+					msg.match[1] = that.stripBotName(robot.name, msg.match[1]);
```

Comment:
@CiaranHannigan 

I see what you're saying but I'm not sure why the user would want to purposefully match the bot's name. Anyways, I added this because in **direct messages** the bot's name is always included. Without a check, it includes the name it in the match. 

Example of 
```
robot: Give me a github user/name and the name of your app.
user: reicruz/node-simple node-app
robot: Your app is named `node-app` and it lives at `robot kjfkdd/kjdfkjd`?
```

I can add a check for `match[1]`, missed that.